src/components/side_panel/chart/chart_with_axis/design_panel.xml

```diff
@@ -18,6 +18,14 @@
             <option value="right">Right</option>
           </select>
         </Section>
+        <Section>
```

Comment:
should probably have a `pt-0` (or be in the same section as the legend), there is a huge padding between the checkbox and the legend select